### ISSUE TITLE
__soapCall satırı hata

### ISSUE THREAD

**Issue by cbinal** [SM-Laravel/Gittigidiyor#2]
$result = $soapClient->__soapCall($method,$soapParams); 
satırından sonra Class devam etmiyor. Hata da vermiyor ama daha ileride gitmiyor.

**Comment by c0b41** [comment]
@cbinal hata bilgisinide paylaşabilirmisin?

**Comment by cbinal** [comment]
maalesef bir hata bilgisi dönmüyor. ama ilgili satırdan sonrasına da geçmiyor.

**Comment by c0b41** [comment]
@cbinal soap kütüphanesi php ini'de aktif mi peki?

**Comment by cbinal** [comment]
evet. zaten SoapClient çağırılırken bir sorun olmuyor. sadece __soapCall çağrılırken sorun oluşuyor. Aynı soapCall metodu class 'dan çıkarılıp çalıştırıldığında sonuç döndürüyor.

**Comment by c0b41** [comment]
@cbinal Bilgi için teşekkürler göz atacağım

**Comment by c0b41** [comment]
@cbinal kontrol gerçekleştirdim ama bir sorun göremedim, örnek bir kod bloğu paylaşırsanız hatayla beraber bakabilirim

**Comment by c0b41** [comment]
sorun çözüldümü, yoksa devam ediyormu?

**Comment by cbinal** [comment]
İlginize teşekkür ederim Ayhan Bey;
soapCall metodunu class içerisinde hala çalıştıramıyorum. Açıkçası detaylı inceleme fırsatı da bulamadım. Acil bitirmem gereken bir proje olduğu için vakit kaybetmeden class olmadan devam ettim. Fakat ilk fırsatta incelemeyi düşünüyorum. Konuyla ilgili görüşlerimi sizinle paylaşmış olacağım.
Saygılarımla
